pkg/kubefedctl/join.go

```diff
@@ -508,6 +523,43 @@ func createServiceAccount(clusterClientset kubeclient.Interface, namespace,
 	default:
 		return saName, nil
 	}
+} // createServiceAccount creates a service account in the cluster associated
```

Comment:
this doesn't look right.
```suggestion
}

// createServiceAccount creates a service account in the cluster associated
```